Some documentation cleanup before 0.7.0.

bdice opened the pull request:
## Description
Minor changes to improve the documentation.

## Types of changes
<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds or improves functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update

## Checklist:
- [x] I have read the [**CONTRIBUTING**](https://github.com/glotzerlab/garnett/blob/master/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] I have updated the documentation (if relevant).
- [ ] I have added tests that cover my changes (if relevant).
- [x] All new and existing tests passed.
- [ ] I have updated the [credits](https://github.com/glotzerlab/garnett/blob/master/doc/credits.rst).
- [ ] I have updated the [Changelog](https://github.com/glotzerlab/garnett/blob/master/changelog.rst).

bdice commented on doc/conf.py:
```diff
@@ -59,15 +59,15 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
```
```suggestion
# The short X.Y.Z version.
```

lyrivera commented on doc/conf.py:
```diff
@@ -59,15 +59,15 @@
 # General information about the project.
 project = 'garnett'
-copyright = '2015-2019, The Regents of the University of Michigan'
+copyright = '2015-2020, The Regents of the University of Michigan'
```
Actually, all the files with the license agreement still have 2019. Can you fix that as part of this PR?

bdice commented:
@lyrivera I updated the copyright years. I also rewrote some of the content in the `examples/` directory. It was written for HOOMD 1.x and no longer worked. Also it was simulating tetrahedra in an example that was labeled "cubes." 🙄🙄🙄
